migration/migrator/migrations/system/20191027013443_separate_shipper_dirs.py

```diff
@@ -0,0 +1,14 @@
+import os
+import shutil
+
+
+def up(config):
+    IN_PROGRESS_PATH = os.path.join(config.submitty['submitty_data_dir'], 'grading')
+
+    os.makedirs(IN_PROGRESS_PATH, exist_ok=True)
+    shutil.chown(IN_PROGRESS_PATH, 'submitty_daemon', 'submitty_daemonphp')
+    os.chmod(IN_PROGRESS_PATH, 0o770)
```

Comment:
This directory is made as part of the regular `SUBMITTY_INSTALL_HELPER.sh` script, so we do not need a migration file for it.